package.json

```diff
@@ -2,10 +2,11 @@
   "name": "@salesforce/sfdx-plugin-lwc-test",
   "description": "Tools for unit testing Lightning web components in a Salesforce DX workspace",
   "version": "0.1.1",
+  "// @oclif/config v1.14.0+ is broken because of https://github.com/oclif/oclif/issues/182": true,
```

Comment:
It sucks that JSON does not support comments. I would suggest adding this comment in the PR description. Anybody looking to know why it wasn't updated will find it there.
```suggestion
```

Reply:
@ravijayaramappa Fair enough. I never think to look there though. 😆 

FWIW it looks like using the `"//"` key would be a safe way to add comments: https://stackoverflow.com/a/14221781